frontend/src/components/MainLayout.tsx

```diff
@@ -2,17 +2,40 @@ import HeaderNav from "./HeaderNav";
 import BottomNav from "./BottomNav";
 import {Outlet} from "react-router-dom";
 import {Container} from "@mui/material";
+import {WebPlaybackSDK} from "react-spotify-web-playback-sdk";
+import {useUserContext} from "../context/userContext";
+import {useCallback} from "react";
 
 export default function MainLayout() {
+    const {user} = useUserContext();
+
+    const getAccessToken = useCallback((callback: (token: string) => void) => {
+        if (!user) {
+            return;
+        }
+
+        callback(user.providerAccessToken);
+    }, [user]);
+
     return (
         <>
-            <HeaderNav/>
+            {user &&
+                /* @ts-ignore */
```

Comment:
:O WHAT?

Reply:
ja I know :( 
das problem ist, dass das WebPlaybackSDK ding aus dem externen package sonst meckert, dass es keine children haben darf. 
Ich glaub das liegt daran, dass ich React 18 hab und das Package auf 17 ausgelegt ist, bin aber nicht sicher. 

Hab bei dem Package auf Github dazu schon ein Issue aufgemacht, in der Hoffnung, dass der Packageowner das fixt und ich das ts-ignore hier wegnehmen kann.
siehe https://github.com/y-hiraoka/react-spotify-web-playback-sdk/issues/2